src/vm-vcpu/src/vcpu/mod.rs

```diff
@@ -15,20 +15,19 @@ use kvm_ioctls::{VcpuExit, VcpuFd, VmFd};
 use vm_device::bus::{MmioAddress, PioAddress};
 use vm_device::device_manager::{IoManager, MmioManager, PioManager};
 use vm_memory::{Address, Bytes, GuestAddress, GuestMemory, GuestMemoryError};
-#[cfg(target_arch = "x86_64")]
-use vm_vcpu_ref::x86_64::gdt::{self, write_idt_value, Gdt, BOOT_GDT_OFFSET, BOOT_IDT_OFFSET};
 use vmm_sys_util::errno::Error as Errno;
 use vmm_sys_util::signal::{register_signal_handler, SIGRTMIN};
 use vmm_sys_util::terminal::Terminal;
 
+use interrupts::*;
 use utils::debug;
+#[cfg(target_arch = "x86_64")]
```

Comment:
Is it the goal that `#[cfg]` is applied only to `use vm_vcpu_ref::x86_64::gdt::{self, write_idt_value, Gdt, BOOT_GDT_OFFSET, BOOT_IDT_OFFSET};` and not to `use vm_vcpu_ref::x86_64::mptable;`. Or both `use`'s should be conditionally compiled?

Reply:
They should both be conditionally compiled. It is going to take some time until we're actually able to receive errors for these kind of things :(. I'll update it.

Reply:
This is fixed in #188 (all the cfgs are in their right place). Is it okay to leave it as is for the time being?